src/web/pages/tasks/icons/__tests__/TaskToolBarIcons.test.tsx

```diff
@@ -5,9 +5,19 @@
 
 import {describe, test, expect, testing} from '@gsa/testing';
 import {screen, rendererWith, fireEvent} from 'web/testing';
+import {vi} from 'vitest';
 import Capabilities from 'gmp/capabilities/capabilities';
 import TaskToolBarIcons from 'web/pages/tasks/icons/TaskToolBarIcons';
 
+vi.mock('web/hooks/useFeatures', () => {
+  return {
+    default: () => ({
+      featureEnabled: (feature: string) =>
+        feature === 'ENABLE_CONTAINER_SCANNING',
+    }),
+  };
+});
+
```

Comment:
useFeatures should not be mocked. You can just pass the desired features to `rendererWith`
```suggestion
```